lib/taskana-core/src/test/java/acceptance/classification/ClassificationServiceAccTest.java

```diff
@@ -0,0 +1,14 @@
+package acceptance.classification;
+
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.TestInstance;
+import org.junit.jupiter.api.TestInstance.Lifecycle;
+
+public class ClassificationServiceAccTest {
+
+  @Nested
+  @TestInstance(Lifecycle.PER_CLASS)
+  class CreateClassificationTest {
+    //
+  }
+}
```

Comment:
yes, but I thought it'd be more convenient for others if this process would be done in one step. Doing this now, would lead to confusion if only CreateClassificationAccTest is in the new structure. If you disagree, I can of course move it over.